tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
         >>> data = [img1, img2]
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
         >>> data = np.array([img1, img2])
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
         >>> wsi_file = 'path/wsi.svs'
```

Comment:
Add an example with all the available options for `CNNPatchPredictor`.

Reply:
Addressed